### ISSUE TITLE
Italia, P 434 Giovanni delle Bande Nere, PPA Full Capability

### ISSUE THREAD

**Issue by Luigi-II** [PygmalionOfCyprus/cmo-db-requests#5095]
### DB Selector

DB3K

### Affected DBID(s)

Ship 3310 3311

### Summary of Changes

- [ ] Rename them as **P 434 Giovanni delle Bande Nere**

### Sources

https://www.seaforces.org/marint/Italian-Navy/Patrol-Vessel/P-434-Giovanni-delle-Bande-Nere.htm
https://en.wikipedia.org/wiki/Italian_offshore_patrol_vessel_Giovanni_delle_Bande_Nere
She is the first PPA with Full Capability (see phased arrays on her bridge): 
![image](https://github.com/user-attachments/assets/92927620-2a75-45c9-af88-b9386305d10d)

**Comment by Tobias02** [comment]
PPA EVO
The new versions of the PPAs will replace the two units sold to Indonesia.
https://svppbellum.blogspot.com/2024/11/marina-militare-italiana-pattugliatori.html

**Comment by Luigi-II** [comment]
> PPA EVO The new versions of the PPAs will replace the two units sold to Indonesia. https://svppbellum.blogspot.com/2024/11/marina-militare-italiana-pattugliatori.html

Yes, I've noticed that. However, at present, it seems that the order of PPA EVO has not been officially signed, I would like to submit the tickets of FREMM EVO first, because they have been ordered. Unfortunately, for various reasons, I was unable to concentrate on those important Italian tickets...
